esque/cli/commands.py

```diff
@@ -1118,3 +1120,134 @@ def ping(state: State, times: int, wait: int):
+
+    Available Handlers:
+
+    \b
+    - kafka
+      <host> -> Used for esque context. If empty, use active context.
+      <path> -> Used as topic name.
+      Supported Parameters:
+        * consumer_group_id
+          Use this consumer group id, when reading from a topic. (Default is "esque-client")
+
+    \b
+    - pipe
+      <host> -> Valid values are "stdin", "stdout" or "stderr".
+      <path> -> No meaning, should stay empty.
+      Supported Parameters:
+        * key_encoding
+          How to embed/extract the message key and value into/from the json objects that are written/read to/from the
+          pipe. Valid values are "base64", "utf-8" and "hex". Use "utf-8" if you know the values are strings and don't
+          mess up the output. Otherwise "base64" is a safer choice. (Default is "utf-8")
+        * value_encoding
+          Same as `key_encoding` but for the values. (Default is "utf-8")
+        * skip_marker
+          Use any value (preferably 1) if you don't want any marker lines to be written between json objects.
+          The marker line "__esque_msg_marker__\\n" is required for esque to be able to **read** from a pipe.
+          Therefore, you can skip the markers in the output when you're not intending to read the piped data back in.
+          You'll even **need** to skip the marker if you want to further process the messages with tools like `jq`.
+
+    \b
+    Available Serializers:
+    - json
+      Supported Parameters:
+        * indent
+          When serializing data into json, use this many spaces to indent (pretty format) the output.
+          By default the serializer uses no indentation and puts everything into one line.
+        * encoding
+          Use this character encoding when transforming the json string into bytes for the message key or value.
+          (Default is "utf-8")
+
+    \b
+    - string
+      Supported Parameters:
+        * encoding
+          Use this character encoding to serialize or deserialize the string to/from the message key or value.
+
+    \b
+    - reg-avro
+      Schema registry based AVRO serializer.
+      Supported Parameters:
+        * schema_registry_uri (required)
+          The url of the schema registry. E.g. https://schema-registry.example.com
+          You can use `esque urlencode https://schema-registry.example.com` to get a representation of that url that's
+          you safe to pass in as query param in the input or output uri.
+        * schema_subject (required only for writing, not for reading)
+          The subject to register new schemas for.
+          A subject is the topic name with "-key" or "-value" as suffix, depending on whether this is the key or value
+          serializer.
+
+    \b
+    EXAMPLES:
+    # Read data from an avro topic
+    esque io -i "kafka+reg-avro:///mytopic?kv__schema_registry_uri=$(esque urlencode https://schema-registry.example.com)"
+
+    \b
+    # Extract and format message value using jq.
```

Comment:
Super small I don't even want to write it 😅 I think if it is a period it should be removed to match the other example descriptions and if it references the `.` you sometimes put after `jq` doesn't there have to be a space in between?

Reply:
No it's fine, you're right, now is the time to fix it :smile: